chainer/optimizer.py

```diff
@@ -652,9 +655,58 @@ def serialize(self, serializer):
             if rule is not None:
                 rule.serialize(serializer[name])
 
+    def loss_scaling(self, interval=1000, scale=None):
+        """Sets loss scaling configurations.
+
+        Args:
+            interval (integer): Number of iterations until scaling factor gets
+                doubled. This is effective when "dynamic" loss scaling is used.
+            scale (float): Loss scaling factor. If None, "dynamic" loss scaling
+                is used, otherwise "static" loss scaling is used.
+        """
+        if scale is None:
+            self._loss_scaling_mode = 'dynamic'
+            if interval < 1:
+                raise ValueError("interval must be greater equal to 1")
+            self._loss_scale = 1.0
+            self._loss_scaling_multiplier = math.pow(2.0, 1.0 / interval)
+            self._loss_scaling_isnan_ever = False
+        else:
+            self._loss_scaling_mode = 'static'
+            if scale <= 0:
+                raise ValueError("loss_scale must be positive number")
+            self._loss_scale = scale
+
     def set_loss_scale(self, loss_scale):
         """Sets loss scaling factor."""
-        self._loss_scale = loss_scale
+        self.loss_scaling('static', loss_scale)
```

Comment:
Seems like it's passing a string according to the old version of `loss_scaling`. Could you change it according to the new version?

Reply:
Ah, sorry, I forgot to change that.

Reply:
Thanks!